xla/backends/gpu/codegen/fusion_emitter.cc

```diff
@@ -266,10 +266,15 @@ absl::StatusOr<llvm::Function*> RemoveUnusedTritonAbiArguments(
           .getCallee();
   llvm::Function* new_function = static_cast<llvm::Function*>(inserted);
 
-  new_function->setCallingConv(impl_fn->getCallingConv());
   new_function->copyMetadata(impl_fn, 0);
   new_function->setAttributes(impl_fn->getAttributes());
 
+  // Set the correct calling convention for the target GPU.
+  // Triton generates PTX_Kernel CC even for AMD, so we need to use
```

Comment:
The comment doesn't make sense to me. This is the fusion emitter that is used if we don't use Triton. It is however used when we compute the reference result for TritonFusionNumericsVerifier. If it is a problem just for TritonFusionNumericsVerifier, I guess the problem should be fixed there?